feat(changelog): postgresql-and-mysql-added-postgresql-pgvector-is-now-a 2023-11-30 (#2484)
scaleway/docs-content: changelog/november2023/2023-11-30-postgresql-and-mysql-added-postgresql-pgvector-is-now-a.mdx

---
title: PostgreSQL pgvector is now available!
status: added
author:
    fullname: 'Join the #database channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-11-30
category: databases
product: postgresql-and-mysql
---

> The pgvector extension allows many AI use cases by turning your PostgreSQL instance into a vector database. Features like embeddings storing or vector similarity search are particularly interesting.

Reviewer: ```suggestion
The `pgvector` extension allows many AI use cases by turning your PostgreSQL instance into a vector database. Features like embeddings storing or vector similarity search are particularly interesting.
```

> The `pgvector` extension allows many AI use cases by turning your PostgreSQL instance into a vector database. Features like embeddings storing or vector similarity search are particularly interesting.

Reviewer 2: ```suggestion
The `pgvector` extension enables many AI use cases by turning your PostgreSQL instance into a vector database. Features like storing embeddings and vector similarity search are particularly interesting.
```
